modin/config/envvars.py

```diff
@@ -154,7 +153,6 @@ class RayRedisPassword(EnvironmentVariable, type=ExactStr):
     """What password to use for connecting to Redis."""
 
     varname = "MODIN_REDIS_PASSWORD"
-    default = secrets.token_hex(32)
```

Comment:
I wonder if we should place `ray_constants.REDIS_DEFAULT_PASSWORD` to the `default` instead of removing this. Probably not explicitly, but through overwriting the `.get()` for lazy `ray` import. 

Current changes make an inconsistent between the password stored in the config and the actual one.

Reply:
Moved to method `_get_default`, it makes possible to import `ray` lazily too

Reply:
if you're going to remove this line, remember to remove `import secrets` as well.

Reply:
`import secrets` was removed already.